src/library-authoring/LibraryAuthoringPage.jsx

```diff
@@ -0,0 +1,135 @@
+// @ts-check
+/* eslint-disable react/prop-types */
+import React, { useEffect } from 'react';
+import { StudioFooter } from '@edx/frontend-component-footer';
+import { FormattedMessage, useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Container, Icon, IconButton, SearchField, Tab, Tabs,
+} from '@openedx/paragon';
+import { InfoOutline } from '@openedx/paragon/icons';
+import {
+  Routes, Route, useLocation, useNavigate, useParams,
+} from 'react-router-dom';
+
+import Loading from '../generic/Loading';
+import SubHeader from '../generic/sub-header/SubHeader';
+import Header from '../header';
+import NotFoundAlert from '../generic/NotFoundAlert';
+import LibraryComponents from './LibraryComponents';
+import LibraryCollections from './LibraryCollections';
+import LibraryHome from './LibraryHome';
+import { useContentLibrary } from './data/apiHook';
+import messages from './messages';
+
+const TAB_LIST = {
+  home: '',
+  components: 'components',
+  collections: 'collections',
+};
+
+const SubHeaderTitle = ({ title }) => (
+  <>
+    {title}
+    <IconButton
+      src={InfoOutline}
+      iconAs={Icon}
+      alt={<FormattedMessage {...messages.headingInfoAlt} />}
+      className="mr-2"
+    />
+  </>
+);
+
+/**
+ * @type {React.FC}
+ */
+const LibraryAuthoringPage = () => {
+  const intl = useIntl();
+  const location = useLocation();
+  const navigate = useNavigate();
+  const [tabKey, setTabKey] = React.useState(TAB_LIST.home);
+  const [searchKeywords, setSearchKeywords] = React.useState('');
+
+  const { libraryId } = useParams();
+
+  const { data: libraryData, isLoading } = useContentLibrary(libraryId);
+
+  useEffect(() => {
+    const currentPath = location.pathname.split('/').pop();
+    if (currentPath && Object.values(TAB_LIST).includes(currentPath)) {
+      setTabKey(currentPath);
+    } else {
+      setTabKey(TAB_LIST.home);
+    }
+  }, [location]);
+
+  if (isLoading) {
+    return <Loading />;
+  }
+
+  if (!libraryId || !libraryData) {
+    return <NotFoundAlert />;
+  }
+
+  /** Handle tab change
+    * @param {string} key
+    */
+  const handleTabChange = (key) => {
+    setTabKey(key);
+    navigate(key);
+  };
+
+  return (
+    <>
+      <Header
+        number={libraryData.version.toString()}
+        title={libraryData.title}
+        org={libraryData.org}
+        contentId={libraryId}
+        isLibrary
+      />
+      <Container size="xl" className="p-4 mt-3">
+        <SubHeader
+          title={<SubHeaderTitle title={libraryData.title} />}
+          subtitle={intl.formatMessage(messages.headingSubtitle)}
+        />
+        <SearchField
+          value={searchKeywords}
+          placeholder={intl.formatMessage(messages.searchPlaceholder)}
+          onChange={(value) => setSearchKeywords(value)}
+          className="w-50"
+        />
```

Comment:
The `onSubmit` still needs to be defined here, as if you press "Enter" it causes an error `onSubmit is not a function`. We can pass in a no-op function `() => {}` since the search is already performed `onChange` so it doesn't fire again.

Reply:
Good catch! Fixed: 388c40e2ffd072d8c1fe770f777d8d7bea9bfe3a